config/webhook.yaml

```diff
@@ -11,7 +11,7 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
+---
```

Comment:
do we need this extra ticks?

Reply:
We need it to construct the yaml file with all the different yamls.